src/finn/custom_op/fpgadataflow/thresholding.py

```diff
@@ -42,6 +42,14 @@ def __init__(self, onnx_node, **kwargs):
 
     def get_nodeattr_types(self):
         my_attrs = {
+            # memory mode for the thresholds
+            # const -- embedded thresholds, default
+            # decoupled -- streaming thresholds with  streamer packaged inside IP
+            "mem_mode": ("s", False, "const", {"const", "decoupled"}),
```

Comment:
`mem_mode` is only used for HLS variant and should therefore not be part of `thresholding.py` but `thresholding_hls.py`.